modules/efs/main.tf

```
@@ -6,7 +6,7 @@ locals {
 }
 
 module "efs" {
-  source = "git::https://github.com/cloudposse/terraform-aws-efs.git?ref=tags/0.21.0"
+  source = "git::https://github.com/cloudposse/terraform-aws-efs.git?ref=tags/0.30.1"
```

nit: We should use the more popular `source`

```suggestion
  source  = "cloudposse/efs/aws"
  version = "0.30.1"
```